lib/Db/BoardMapper.php

```diff
@@ -66,7 +66,7 @@ public function __construct(
 	 * @throws DoesNotExistException
 	 */
 	public function find($id, $withLabels = false, $withAcl = false) {
-		$sql = 'SELECT id, title, owner, color, archived, deleted_at, last_modified FROM `*PREFIX*deck_boards` ' .
+		$sql = 'SELECT id, title, owner, color, archived, deleted_at, last_modified, cover_images FROM `*PREFIX*deck_boards` ' .
```

Comment:
I'm not entirely sure if we should rather make this a user setting. In any case we should reuse the ConfigService for this instead of adding more fields to the board table to keep it flexible.